py/server/deephaven/table.py

```diff
@@ -251,6 +251,66 @@ def with_filters(self, filters: Union[str, Filter, Sequence[str], Sequence[Filte
         except Exception as e:
             raise DHError(e, "with_filters operation on RollupTable failed.") from e
 
+    def layout_hints(self, front: Union[str, List[str]] = None, back: Union[str, List[str]] = None,
+                     freeze: Union[str, List[str]] = None, hide: Union[str, List[str]] = None,
+                     column_groups: List[dict] = None, search_display_mode: SearchDisplayMode = None) -> Table:
+        """ Sets layout hints on the RollupTable
+
+        Args:
+            front (Union[str, List[str]]): the columns to show at the front.
+            back (Union[str, List[str]]): the columns to show at the back.
+            freeze (Union[str, List[str]]): the columns to freeze to the front.
```

Comment:
This leaves a lot of questions for me:
1. Is this separate from `front`?
2. Do the frozen columns appear before the front columns?
3. Do the frozen columns need to be listed in the front columns?

Reply:
I just copied these from the existing `layout_hints` method that is already on `Table`. Should I make updates there as well?
I'm just trying to get this supposed Enterprise blocker completed. Didn't expect/need to refactor a whole bunch of APIs, as this is supposed to be a hotfix - trying to change as little as possible, while just exposing this functionality in Python (though they _did_ only ask about Groovy...)